webpack/components/AnsibleHostDetail/components/RolesTab/index.js

```diff
@@ -33,9 +37,17 @@ const RolesTab = ({ hostId, history, canEditHost }) => {
       onClick={() => setAssignModal(true)}
       aria-label="edit ansible roles"
     >
-      {__('Assign Ansible roles')}
+      {__('Assign Ansible roles directly to the host')}
     </Button>
   ) : null;
+
+  const url = hostId && foremanUrl(`/api/v2/hosts/${hostId}/ansible_roles`);
+  const { response: allAnsibleRoles } = useAPI('get', url);
+  const emptyStateDescription =
+    allAnsibleRoles.length > 0 ? (
+      <Link to="/Ansible/roles/all">{__('view inherited Ansible roles')}</Link>
```

Comment:
Nit: Link text is fine, but since user is on Ansible page, the button can be simply `View inherited roles`,
it's clear that we mean roles from Ansible

Reply:
How about:
```js
allAnsibleRoles.length > 0 && <Link to="/Ansible/roles/all">{__('view inherited Ansible roles')}</Link>
```